theming.theme.inc

```diff
@@ -133,9 +133,9 @@ function theming_form_element($variables) {
   if (!empty($element['#description'])) {
     $output .= '<div' . drupal_attributes($element['#description_attributes']) . '>' . $element['#description'] . "</div>\n";
   }
-
-  $output .= "</div>\n";
-
+  if (empty($element['#wrapper_enabled'])) {
```

Comment:
Why if empty? Don’t we want a wrapper `if ($element['#wrapper_enabled'])`?

Reply:
Ah yes. Seems I forgot to rewrite this condition after I had reintroduced the flag. Fixed in a new commit.